tests/plotting/test_plotting.py

```diff
@@ -1239,6 +1239,14 @@ def test_plot_texture():
     plotter.show()
 
 
+def test_plot_numpy_texture():
```

Comment:
This mark means skipping the test when the environment doesn't have `imageio`.
If the addition is OK, please Apply.
```suggestion
@pytest.mark.skipif(not HAS_IMAGEIO, reason="Requires imageio")
def test_plot_numpy_texture():
```